libbuteosyncfw/profile/SyncSchedule.h

```diff
@@ -170,13 +168,13 @@ class SyncSchedule
      * \return True if schedule is controlled by a external process. External process will control the sync,
      * buteo schedule is disabled in this case.
      */
-    bool externalScheduleEnabled() const;
+    bool alwaysOnEnabled() const;
 
     /*! \brief Sets if schedule is controlled by a external process.
      *
      * \param aEnabled Specify if schedule is contolled by a external process. If set to true, buteo schedule will be set to disabled.
      */
-    void setExternalScheduleEnabled(bool aEnabled);
+    void setAlwaysOnEnabled(bool aEnabled);
```

Comment:
To me the previous naming is preferable as it better reflects the intentions of setting this property; setting "always on" sounds more like buteo is expected to sync this schedule more often. Maybe setSyncExternally would hint more at the fact that the internal buteo schedule would automatically be disabled than setExternalScheduleEnabled.

As mentioned elsewhere it seems odd to set this property on the schedule, as setting this property means the schedule is not necessary at all, so perhaps it only needs to be set in SyncProfile and not SyncSchedule?